### ISSUE TITLE
redirection to /unauthorised

### ISSUE THREAD

**Issue by stiliajohny** [rmountjoy92/DashMachine#207]
After I deploy dashmachine ( all different ways, docker/python/k8s) I am always getting redirected to the `/unauthorized` page. 
The page doesn't resolve ( getting a 403 ) 
Is this a bug? 
Can we have the default redirect to `/login`?

**Comment by dineshkumar181094** [comment]
yes the redirection happens of HTTP not on HTTPS ultimately leading to the issue. Need to enter https manually all the time.

**Comment by stiliajohny** [comment]
will try that ! thanks, @dineshkumar181094

**Comment by FlyingEagle** [comment]
still happening ... is there a new version in sight or do I a´have to look for an other solution better?
